repository/src/main/java/org/pentaho/platform/security/userroledao/jackrabbit/AbstractJcrBackedUserRoleDao.java

```diff
@@ -841,9 +842,13 @@ private RepositoryFile createUserHomeFolder( ITenant theTenant, String username,
           JcrRepositoryFileUtils.getFileByAbsolutePath( session, ServerRepositoryPaths.getUserHomeFolderPath(
               theTenant, username ), pathConversionHelper, lockHelper, false, null );
       if ( userHomeFolder == null ) {
+        RepositoryFile.Builder newFolder = new RepositoryFile.Builder( username ).folder( true );
+        Boolean hideUserHomeFolder = "true".equals(
+          PentahoSystem.get( ISystemConfig.class ).getProperty( "system.hideUserHomeFolderOnCreate" ) );
```

Comment:
Maybe add a "toLowerCase()" to the value obtained from getProperty, if not null, and create a constant to hold the property name.

Reply:
ok